packages/qwik-nx/src/plugins/qwik-nx-vite.plugin.spec.ts

```diff
@@ -1,11 +1,14 @@
 import { workspaceRoot } from '@nrwl/devkit';
 import { join } from 'path';
-import { qwikNxVite, QwikNxVitePluginOptions } from './qwik-nx-vite.plugin';
+import { qwikNxVite } from './qwik-nx-vite.plugin';
+import { QwikNxVitePluginOptions } from './models/qwik-nx-vite';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const fileUtils = require('nx/src/project-graph/file-utils');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const fs = require('fs');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const getProjectDependenciesModule = require('./utils/get-project-dependencies');
 
 const workspaceConfig1 = {
```

Comment:
in general I would avoid using consts outside of a test function (because they could be modified and create unexpected dependencies between test)

Prefer using a function that returns a new const every time you call it

makes sense?

Reply:
Totally, thanks for catching this!